include/retdec/cpdetect/compiler_detector/heuristics/pe_heuristics.h

```diff
@@ -40,12 +46,18 @@ class PeHeuristics : public Heuristics
 
 		/// @name Heuristics for detection of used compiler or packer
 		/// @{
+        int32_t get_int32_unaligned(const uint8_t * codePtr);
+        const uint8_t * skip_NOP_JMP8_JMP32(const uint8_t * codeBegin, const uint8_t * codePtr, const uint8_t * codeEnd, size_t maxCount);
+		void getHeaderStyleHeuristics();
```

Comment:
Nitpick: Inconsistent use of tabs vs spaces (see the diff on GitHub, which makes it clear).